src/interface/src/app/map/map.helper.ts

```diff
@@ -17,11 +17,38 @@ export function checkIfAreaInBoundaries(
   boundaries: Feature
 ): boolean {
   let overlappingAreas = area.features.map((feature) => {
+    // if we want to allow multipolygons we need to transform them to polygons
+    // to validate if in area.
+    // if (feature.geometry.type === 'MultiPolygon') {
+    //   return checkIfMultipolygonOverlaps(
+    //     feature as Feature<MultiPolygon>,
+    //     boundaries
+    //   );
+    // }
     return !booleanWithin(feature, boundaries);
   });
   return !overlappingAreas.some((overlap) => overlap);
 }
 
+// @ts-ignore
+function checkIfMultipolygonOverlaps(
```

Comment:
I left this here in case we fix the API and want to validate multipolygons in the future